packages/frontend/ci/configFromEnvironment.js

```diff
@@ -30,7 +30,7 @@ module.exports = {
     SENTRY_RELEASE:
       process.env.SENTRY_RELEASE ||
       (parseBooleanFromShell(process.env.RENDER) &&
-          `render:${process.env.RENDER_SERVICE_NAME}:${process.env.RENDER_GIT_BRANCH}:${process.env.RENDER_GIT_COMMIT}`),
+          `render:${process.env.RENDER_SERVICE_NAME}:${process.env.RENDER_GIT_BRANCH.replace(/\//g, '-')}:${process.env.RENDER_GIT_COMMIT}`),
```

Comment:
The `/` character is invalid in Sentry identifiers, causing Render `near-wallet` deployments to fail. Only affects `testnet` PR previews.